src/plugins/intel_npu/src/CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ add_subdirectory(utils)
 
 add_subdirectory(al)
 
+if (NOT BUILD_COMPILER_FOR_DRIVER)
```

Comment:
We might find other reasons to exclude this in the future. Should we have a dedicated option for it that can be disabled independent of other options? We can mark ENABLE_DRIVER_COMPILER_ADAPTER and ENABLE_ZEROAPI_BACKEND as dependent of this.

Reply:
done